doc/tools.rst

```diff
@@ -16,5 +16,17 @@ The standard distribution ships with the following tools:
 - | `C2nim <https://github.com/nim-lang/c2nim/blob/master/doc/c2nim.rst>`_
   | C to Nim source converter. Translates C header files to Nim.
 
+- | `niminst <https://nim-lang.org/docs/niminst.html>`_
+  | niminst is a tool to generate an installer for a Nim program.
+
 - | `nimgrep <nimgrep.html>`_
   | Nim search and replace utility.
+
+- | nimpretty
+  | ``nimpretty`` is a Nim source code beatifier,
+    to format code according to the official style guide.
+
+- | REPL (Experimental)
```

Comment:
Don't document ``nim secret`` please, we are currently not working on this.

Reply:
Done :grey_exclamation: